src/components/inputs/TokenSearchInput.vue

```diff
@@ -48,106 +130,10 @@
     <teleport to="#modal">
       <SelectTokenModal
         v-if="selectTokenModal"
-        :excluded-tokens="modelValue"
+        :excluded-tokens="compact([...modelValue, veBalTokenInfo?.address])"
```

Comment:
^ removing it from the select modal and using `compact` since the address might be undefined initially.